apps/src/templates/VersionRow.jsx

```diff
@@ -35,35 +37,44 @@ export default class VersionRow extends React.Component {
         </button>
       );
     } else {
-      button = [
+      buttons.push(
+        <button
+          key={buttons.length}
+          type="button"
+          className="btn-info"
+          onClick={this.props.onChoose}
+        >
+          {msg.restoreThisVersion()}
+        </button>
+      );
+    }
+
+    if (!this.props.isActive) {
+      buttons.push(
         <a
-          key={0}
+          key={buttons.length}
           href={
             location.origin +
             location.pathname +
-            '?version=' +
-            this.props.versionId
+            (this.props.isLatest ? '' : '?version=' + this.props.versionId)
           }
           target="_blank"
           rel="noopener noreferrer"
         >
           <button type="button" className="version-preview">
             <i className="fa fa-eye" />
           </button>
-        </a>,
-        <button
-          type="button"
-          key={1}
-          className="btn-info"
-          onClick={this.props.onChoose}
-        >
-          {msg.restoreThisVersion()}
-        </button>
-      ];
+        </a>
+      );
     }
 
     return (
-      <tr className="versionRow">
+      <tr
+        className={[
```

Comment:
Could use `classnames` library here with `classnames('versionRow', {'activeVersion': this.props.isActive})`